### PR TITLE
add plan diff at the bottom for plan diff view

### PR DESCRIPTION
**Pull request by keanecjy**
Currently:
Version does not change

### CODE REVIEW

File: TourMate/TourMate/Frontend/Plan/Views/PlanDiffView/PlanDiffView.swift [hunk 1]
```diff
@@ -9,22 +9,44 @@ import SwiftUI
...
+    @State private var rightVersion: Int
+
+    let planDiffUtil: PlanDiffUtil
 
     init(planViewModel: PlanViewModel<T>) {
-        self.viewModel = planViewModel
+        let currentVersion = planViewModel.versionNumber
+
+        self._leftVersion = State(wrappedValue: planViewModel.isLatest ? currentVersion : currentVersion - 1)
+        self._rightVersion = State(wrappedValue: planViewModel.versionNumber)
```
Comment (Vielheim) [inline TourMate/TourMate/Frontend/Plan/Views/PlanDiffView/PlanDiffView.swift:28]: By default when we enter the PlanDiff the viewmodel is in the latest version, so both left and right views are currently using the latest version? I think the original logic was okay? Unless something there's something new

**Review by Vielheim — APPROVED** [review]
LGTM, just small comment about the versioning

File: TourMate/TourMate/Frontend/Plan/Views/PlanDiffView/PlanDiffView.swift (hunk 1 above)
Comment (keanecjy) [reply TourMate/TourMate/Frontend/Plan/Views/PlanDiffView/PlanDiffView.swift:28]: oh previously i will set it to use the previous version. But this change somehow does not set the version

Comment (keanecjy) [reply TourMate/TourMate/Frontend/Plan/Views/PlanDiffView/PlanDiffView.swift:28]: Ok ya my bad haha the condition should be the other way around

Comment (keanecjy) [reply TourMate/TourMate/Frontend/Plan/Views/PlanDiffView/PlanDiffView.swift:28]: The comments view model seem to be re-creating itself though.. Trying to fix

**Pull request merged**